cryptographic_estimators/SDEstimator/SDAlgorithms/stern.py

```diff
@@ -28,24 +36,22 @@
 class Stern(SDAlgorithm):
     def __init__(self, problem: SDProblem, **kwargs):
         """
-        Construct an instance of Stern's estimator [Ste88]_, [BLP08]_.
+        Construct an instance of Stern's estimator.
```

Comment:
citation completely deleted.

Reply:
This is really bad.

Thanks for the catch.